sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPluginInternal.scala

```diff
@@ -333,7 +334,12 @@ object ScalaJSPluginInternal {
 
           val sourceMapFile = FileVirtualJSFile(output).sourceMapFile
           Attributed.blank(output).put(scalaJSSourceMap, sourceMapFile)
-        }.tag(usesLinkerTag)
+        }
+
+        linkerTask
+          .tag(usesLinkerTag)
+          .tag(scalaJSLinkTag)
```

Comment:
I believe `.tag()` takes varargs, both in sbt 0.13.x and 1.x. If that's true, I suggest keeping the old formatting and just calling
```scala
}.tag(usesLinkerTag, scalaJSLinkTag)
```